apps/frontend/src/components/auth/Login.tsx

```diff
@@ -16,7 +16,7 @@ export function Login() {
     }
   }, [route, navigate, from]);
   return (
-    <View className="auth-wrapper">
+    <View className="auth-wrapper" style={{ display: 'flex', alignItems: 'center', justifyContent: 'center', minHeight: 'calc(100vh - 64px)' }}>
```

Comment:
❓ Do these need to be inline styles? Is there a way to express this via a Chakra Box? That would be better as it is consistent with our styling library.